workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -75,6 +75,13 @@ HOSTED_PROMETHEUS_TOKEN: <truncated>
 HCP_NAMESPACE: ${HCP_NAMESPACE}
 MGMT_WORKER_NODES: ${MGMT_WORKER_NODES}
 EOF
+
+  if [[ ${WORKLOAD} =~ "index" ]]; then
+    cat << EOF
+elapsed: "20m:"
+EOF
+  fi
+  
```

Comment:
What's this for?

Reply:
kube-burner set this variable at workload execution to limit the duration of certain metrics like [this](https://github.com/cloud-bulldozer/e2e-benchmarking/blob/master/workloads/kube-burner-ocp-wrapper/metrics-profiles/mc-metrics.yml#L16) so I am hard coding here to 20m. Yes I should remove the `:`.

Reply:
why 20m? what's wrong with the original .elapsed? if you don't need it anymore, set everything to 20m to avoid confusion

Reply:
I guess there is confusion here, idea is to call only index without workload so I assume elapsed will not be set by kube-burner based on [this](https://github.com/cloud-bulldozer/kube-burner/blob/71a4ba5ec036650cf9827235746dd84cbf73cb74/docs/observability/metrics.md?plain=1#L73). there is no workload executed by kube-burner at first place to get the duration of job. 
If my understanding is wrong, I can remove this elapsed variable from here.

Reply:
oh, that's right, elapsed won't be set unless your using this profile from a benchmark. Now I wonder...
These lines are correct?
```shell
  if [[ ${WORKLOAD} =~ "index" ]]; then
    cat << EOF
elapsed: "20m"
EOF
  fi
```

That statement print `elapsed: "20"` in the standard output, shouldn't be `export elapsed=20m`?

Reply:
right, replaced it.